docs/specs/shapes.md

```diff
@@ -663,3 +663,28 @@ separately, $start$ and $end$ being applied to each shape.
 * When `m` has a value of `2` (Sequential), all the shapes MUST be considered
 as following each other in render order.  $start$ and $end$ refer to the whole
 length created by concatenating each shape.
+
+
+<h3 id="trim-path">Rounded Corners</h3>
```

Comment:
Can we include the rendering algorithm as well? Particularly how it's different from rectangle roundness (see https://github.com/lottie/lottie-spec/issues/82)

Reply:
id is wrong